### DISCUSSION TITLE
Suggestion - Edit machine tags

### DISCUSSION THREAD

**Discussion by aggietallboy** [open-amt-cloud-toolkit/open-amt-cloud-toolkit#218] [Ideas]
Howdy all!
I'm really liking the system as I've been using it more.
I'd like to be able to edit machine tags without having to create a whole different profile to do so.
Being able to add/remove tags from the device list would be very helpful (as we've already got filtering based on them!)

**Comment by graikhel-intel** [comment]
We have an API to do this but need to add it to sample-web-ui. I have created an issue to track this request https://github.com/open-amt-cloud-toolkit/sample-web-ui/issues/1245.
